src/test/ui/issues/issue-3044.stderr

```diff
@@ -25,10 +15,10 @@ LL |     fn fold<B, F>(mut self, init: B, mut f: F) -> B
 help: provide the argument
    |
 LL ~     needlesArr.iter().fold(|x, y| {
+LL +         x
```

Comment:
Somewhat confused why this is `+` and not `|` 🤔

Reply:
it has to do with suggestions and newlines and whitespace i think, though i don't think the actual suggestion is wrong, just the rendering...?